src/globalComponents/typesAndClasses/Map.js

```diff
@@ -0,0 +1,5 @@
+//@flow strict

```

Comment:
This is a duplicate of the ArenaType class I think

Reply:
Ah, I couldn't find one. I'll update it.